s3/replication/common/src/templates/replication_job_template_for_manager.json

```diff
@@ -0,0 +1,35 @@
+{
+  "ACL": "base64-encoded-acl-json",
+  "Bucket-Name": "sourcebucket",
```

Comment:
since its a template, use placeholder names, see other template files.

Reply:
This comment is pending. Example see https://github.com/Seagate/cortx-multisite/blob/main/s3/replication/common/src/templates/replication_job_template.json#L14-L17

Reply:
done